lib/Crust/Request.pm6

```diff
@@ -84,7 +84,7 @@ method header(Str $name) {
 method content() {
     # TODO: we should support buffering in Crust layer
     my $input = $!env<p6sgi.input>;
-    $input.seek(0,0); # rewind
+    $input.seek(0); # rewind
```

Comment:
If `$!env<p6sgi.input>` is not `IO::Handle` instance such as `IO::Blob` instance, then seek is failed.